src/main/java/intefaceAdapters/userLogin/FindUser.java

```diff
@@ -0,0 +1,39 @@
+package intefaceAdapters.userLogin;
+
+import useCases.userLogin.UserLoginDsGateway;
+
+import java.io.*;
+import java.util.HashMap;
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+public class FindUser implements UserLoginDsGateway {
+    private final Map<String, String> accounts = new HashMap<>();
+
+
+    public FindUser(String csvPath) throws IOException {
+        File csvFile = new File(csvPath);
+        Map<String, Integer> headers = new LinkedHashMap<>();
+        headers.put("email", 0);
+        headers.put("password", 1);
+        headers.put("name", 2);
+        BufferedReader reader = new BufferedReader(new FileReader(csvFile));
+        reader.readLine(); // skip header
+
+        String row;
+        while ((row = reader.readLine()) != null) {
+            String[] col = row.split(",");
+            String email = String.valueOf(col[headers.get("email")]);
+            String password = String.valueOf(col[headers.get("password")]);
+            accounts.put(email,password);
+        }
```

Comment:
This piece of code is too long to be in a constructor for a class. I recommend keeping the constructor clean and as simple as possible by extracting the functionality to a helper method.